nucypher/policy/maps.py

```diff
@@ -54,7 +53,7 @@ class NowhereToBeFound(RestMiddleware.NotFound):
 
     ursula_and_kfrag_payload_splitter = BytestringSplitter(
         (to_checksum_address, ETH_ADDRESS_BYTE_LENGTH),
-        (UmbralMessageKit, VariableLengthBytestring),
+        (MessageKit, VariableLengthBytestring),
     )
 
     @classmethod
```

Comment:
`bob` parameter on L63 is no longer used and can be removed - was introduced as part of #2773.